<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KeyGenClass</class>
 <widget class="QMainWindow" name="KeyGenClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>618</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>428</width>
    <height>264</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>618</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>&quot;Роза&quot; Генератор секретного ключа </string>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="leftMargin">
     <number>4</number>
    </property>
    <property name="topMargin">
     <number>1</number>
    </property>
    <property name="rightMargin">
     <number>1</number>
    </property>
    <property name="bottomMargin">
     <number>1</number>
    </property>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>3</number>
      </property>
      <widget class="QWidget" name="page">
       <widget class="QLineEdit" name="lineEditKeyPath">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>540</y>
          <width>401</width>
          <height>22</height>
         </rect>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButtonW1Cancel">
        <property name="geometry">
         <rect>
          <x>900</x>
          <y>580</y>
          <width>80</width>
          <height>23</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>22</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton 

{

   border-style: solid;
 
   border-radius: 10;

    border-top          : 1px solid rgba(200, 200, 200, 255);

    border-left         : 1px solid rgba(200, 200, 200, 255);

    border-right        : 2px solid rgba(172, 172, 172);

    border-bottom       : 2px solid rgba(172, 172, 172);

   /*
    background-color    : qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 rgba(198, 198, 198), stop:1 rgba(225, 225, 225, 255));
*/
background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #ffffff, stop: 0.1 #eaeaea, stop: 0.49 #eaeaea, stop: 0.5 #eaeaea, stop: 1 #ffffff);
font-size: 16px;
}



QPushButton:hover 

{
/*
    background-color    : qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 rgba(225, 225, 225, 255), stop:1 rgba(163, 163, 163));
*/
    background-color    : qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 rgba(196, 228, 255), stop:1 rgba(225, 225, 225, 255));
}



QPushButton:pressed 

{

    padding-top         : 2px;

    padding-left        : 2px;

    border-top          : 2px solid rgba(117, 117, 117, 255);

    border-left         : 2px solid rgba(117, 117, 117, 255);

    border-right        : 1px solid rgba(200, 200, 200, 255);

    border-bottom       : 1px solid rgba(200, 200, 200, 255);

    background-color    : rgba(172, 172, 172);

}



QPushButton:disabled 

{

    background-color    : qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 rgba(225, 225, 255, 255), stop:1 rgba(172, 172, 172));

}</string>
        </property>
        <property name="text">
         <string>Отмена</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelKeyPathName">
        <property name="geometry">
         <rect>
          <x>8</x>
          <y>540</y>
          <width>301</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Путь к каталогу хранения ключа:</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButtonW1Next">
        <property name="geometry">
         <rect>
          <x>800</x>
          <y>580</y>
          <width>80</width>
          <height>23</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>22</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>-1</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton 

{

   border-style: solid;
 
   border-radius: 10;

    border-top          : 1px solid rgba(200, 200, 200, 255);

    border-left         : 1px solid rgba(200, 200, 200, 255);

    border-right        : 2px solid rgba(172, 172, 172);

    border-bottom       : 2px solid rgba(172, 172, 172);

   /*
    background-color    : qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 rgba(198, 198, 198), stop:1 rgba(225, 225, 225, 255));
*/
background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #ffffff, stop: 0.1 #eaeaea, stop: 0.49 #eaeaea, stop: 0.5 #eaeaea, stop: 1 #ffffff);
font-size: 16px;
}



QPushButton:hover 

{
/*
    background-color    : qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 rgba(225, 225, 225, 255), stop:1 rgba(163, 163, 163));
*/
    background-color    : qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 rgba(196, 228, 255), stop:1 rgba(225, 225, 225, 255));
}



QPushButton:pressed 

{

    padding-top         : 2px;

    padding-left        : 2px;

    border-top          : 2px solid rgba(117, 117, 117, 255);

    border-left         : 2px solid rgba(117, 117, 117, 255);

    border-right        : 1px solid rgba(200, 200, 200, 255);

    border-bottom       : 1px solid rgba(200, 200, 200, 255);

    background-color    : rgba(172, 172, 172);

}



QPushButton:disabled 

{

    background-color    : qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 rgba(225, 225, 255, 255), stop:1 rgba(172, 172, 172));

}</string>
        </property>
        <property name="text">
         <string>Далее &gt;</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButtonSelectKeyPath">
        <property name="geometry">
         <rect>
          <x>715</x>
          <y>540</y>
          <width>23</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">text-align:left;
background-image: url(&quot;Image/open-folder-yellow.png&quot;);
background-repeat: no-repeat;
background-position: left 
</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>0</y>
          <width>501</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>TimesNewRoman</family>
          <pointsize>-1</pointsize>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">font-family:TimesNewRoman;
font-style: normal;
font-weight: bold;

font-size:15px;
color:#1B1BB3;</string>
        </property>
        <property name="text">
         <string>Выберите каталог хранения секретного ключа.</string>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>40</y>
          <width>991</width>
          <height>481</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">text-align:left;
background-image: url(&quot;Image/digkey.jpg&quot;);
background-repeat: no-repeat;
background-position: left 
</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_3">
       <widget class="QPushButton" name="pushButtonW2Next">
        <property name="geometry">
         <rect>
          <x>900</x>
          <y>580</y>
          <width>80</width>
          <height>23</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>22</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>-1</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton 

{

   border-style: solid;
 
   border-radius: 10;

    border-top          : 1px solid rgba(200, 200, 200, 255);

    border-left         : 1px solid rgba(200, 200, 200, 255);

    border-right        : 2px solid rgba(172, 172, 172);

    border-bottom       : 2px solid rgba(172, 172, 172);

   /*
    background-color    : qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 rgba(198, 198, 198), stop:1 rgba(225, 225, 225, 255));
*/
background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #ffffff, stop: 0.1 #eaeaea, stop: 0.49 #eaeaea, stop: 0.5 #eaeaea, stop: 1 #ffffff);
font-size: 16px;
}



QPushButton:hover 

{
/*
    background-color    : qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 rgba(225, 225, 225, 255), stop:1 rgba(163, 163, 163));
*/
    background-color    : qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 rgba(196, 228, 255), stop:1 rgba(225, 225, 225, 255));
}



QPushButton:pressed 

{

    padding-top         : 2px;

    padding-left        : 2px;

    border-top          : 2px solid rgba(117, 117, 117, 255);

    border-left         : 2px solid rgba(117, 117, 117, 255);

    border-right        : 1px solid rgba(200, 200, 200, 255);

    border-bottom       : 1px solid rgba(200, 200, 200, 255);

    background-color    : rgba(172, 172, 172);

}



QPushButton:disabled 

{

    background-color    : qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 rgba(225, 225, 255, 255), stop:1 rgba(172, 172, 172));

}</string>
        </property>
        <property name="text">
         <string>Далее &gt;</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>406</x>
          <y>30</y>
          <width>591</width>
          <height>71</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Tahoma</family>
          <pointsize>28</pointsize>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">font-family:Tahoma;
font-style: normal;
font-size: 28pt;
font-weight: bold;


color:blue;</string>
        </property>
        <property name="text">
         <string>Генерация секретного ключа</string>
        </property>
       </widget>
       <widget class="QProgressBar" name="progressBarW3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>498</y>
          <width>971</width>
          <height>41</height>
         </rect>
        </property>
        <property name="value">
         <number>24</number>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>490</x>
          <y>160</y>
          <width>451</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>25</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Нажимайте любые клавиши</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelW3EndTime">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>544</y>
          <width>141</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Осталось 4:59</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelAnimation">
        <property name="geometry">
         <rect>
          <x>3</x>
          <y>10</y>
          <width>400</width>
          <height>400</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_4">
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>100</y>
          <width>661</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>25</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Введите пароль защиты секретного ключа.</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButtonW3Next">
        <property name="geometry">
         <rect>
          <x>900</x>
          <y>570</y>
          <width>80</width>
          <height>23</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>22</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>-1</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton 

{

   border-style: solid;
 
   border-radius: 10;

    border-top          : 1px solid rgba(200, 200, 200, 255);

    border-left         : 1px solid rgba(200, 200, 200, 255);

    border-right        : 2px solid rgba(172, 172, 172);

    border-bottom       : 2px solid rgba(172, 172, 172);

   /*
    background-color    : qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 rgba(198, 198, 198), stop:1 rgba(225, 225, 225, 255));
*/
background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #ffffff, stop: 0.1 #eaeaea, stop: 0.49 #eaeaea, stop: 0.5 #eaeaea, stop: 1 #ffffff);
font-size: 16px;
}



QPushButton:hover 

{
/*
    background-color    : qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 rgba(225, 225, 225, 255), stop:1 rgba(163, 163, 163));
*/
    background-color    : qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 rgba(196, 228, 255), stop:1 rgba(225, 225, 225, 255));
}



QPushButton:pressed 

{

    padding-top         : 2px;

    padding-left        : 2px;

    border-top          : 2px solid rgba(117, 117, 117, 255);

    border-left         : 2px solid rgba(117, 117, 117, 255);

    border-right        : 1px solid rgba(200, 200, 200, 255);

    border-bottom       : 1px solid rgba(200, 200, 200, 255);

    background-color    : rgba(172, 172, 172);

}



QPushButton:disabled 

{

    background-color    : qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 rgba(225, 225, 255, 255), stop:1 rgba(172, 172, 172));

}</string>
        </property>
        <property name="text">
         <string>Далее &gt;</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>12</y>
          <width>291</width>
          <height>291</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-image:url(&quot;Image/Lock.png&quot;);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEditW4Password">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>336</y>
          <width>301</width>
          <height>20</height>
         </rect>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEditW4PasswordReply">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>392</y>
          <width>301</width>
          <height>20</height>
         </rect>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>371</x>
          <y>313</y>
          <width>81</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Пароль</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>368</y>
          <width>321</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Подтверждение пароля</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_5">
       <widget class="QPushButton" name="pushButtonW5PrintZayavka">
        <property name="geometry">
         <rect>
          <x>410</x>
          <y>580</y>
          <width>181</width>
          <height>23</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>22</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>-1</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton 

{

   border-style: solid;
 
   border-radius: 10;

    border-top          : 1px solid rgba(200, 200, 200, 255);

    border-left         : 1px solid rgba(200, 200, 200, 255);

    border-right        : 2px solid rgba(172, 172, 172);

    border-bottom       : 2px solid rgba(172, 172, 172);

   /*
    background-color    : qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 rgba(198, 198, 198), stop:1 rgba(225, 225, 225, 255));
*/
background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #ffffff, stop: 0.1 #eaeaea, stop: 0.49 #eaeaea, stop: 0.5 #eaeaea, stop: 1 #ffffff);
font-size: 16px;
}



QPushButton:hover 

{
/*
    background-color    : qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 rgba(225, 225, 225, 255), stop:1 rgba(163, 163, 163));
*/
    background-color    : qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 rgba(196, 228, 255), stop:1 rgba(225, 225, 225, 255));
}



QPushButton:pressed 

{

    padding-top         : 2px;

    padding-left        : 2px;

    border-top          : 2px solid rgba(117, 117, 117, 255);

    border-left         : 2px solid rgba(117, 117, 117, 255);

    border-right        : 1px solid rgba(200, 200, 200, 255);

    border-bottom       : 1px solid rgba(200, 200, 200, 255);

    background-color    : rgba(172, 172, 172);

}



QPushButton:disabled 

{

    background-color    : qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 rgba(225, 225, 255, 255), stop:1 rgba(172, 172, 172));

}</string>
        </property>
        <property name="text">
         <string>Закрыть</string>
        </property>
       </widget>
       <widget class="QTextEdit" name="textEditW5Message">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>16</y>
          <width>971</width>
          <height>551</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"> QTextEdit {
/* 
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                   stop: 0 #E0E0E0, stop: 1 #FFFFFF);
*/
background-color:white;
 border: 2px solid gray;
 border-radius: 5px;
 margin-top: 1ex; /* leave space at the top for the title */
}</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>lineEditKeyPath</tabstop>
  <tabstop>pushButtonSelectKeyPath</tabstop>
  <tabstop>pushButtonW1Next</tabstop>
  <tabstop>pushButtonW1Cancel</tabstop>
  <tabstop>pushButtonW2Next</tabstop>
  <tabstop>lineEditW4Password</tabstop>
  <tabstop>lineEditW4PasswordReply</tabstop>
  <tabstop>pushButtonW3Next</tabstop>
  <tabstop>textEditW5Message</tabstop>
  <tabstop>pushButtonW5PrintZayavka</tabstop>
 </tabstops>
 <resources>
  <include location="KeyGen.qrc"/>
 </resources>
 <connections/>
</ui>
